examples/va/nfv/nfv-ovs-dpdk-sriov-hci/README.md

```diff
@@ -0,0 +1,40 @@
+# Hyperconverged OpenStack and Ceph
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/commit/aa63bf3931f74722dd48af8a0914233b2b384330) on Dec 19th, 2023**
```

Comment:
When this is tested, this line should be updated to reflect which commit of OpenStack Operator was used.

Reply:
ok, i will test again and update it. Thanks